core/tests/protractor_desktop/additionalEditorAndPlayerFeatures.js

```diff
@@ -87,6 +87,22 @@ describe('Full exploration editor', function() {
     explorationPlayerPage.reportExploration();
   });
 
+  it('should let learners suggest changes to an exploration', function() {
+    users.createUser('creator@editorAndPlayer.com', 'creatorEditorAndPlayer');
+    users.login('creator@editorAndPlayer.com');
+    workflow.createAndPublishExploration('Adding Fractions', 'Mathematics',
+      'Let us learn how to add fractions', 'English');
+    users.logout();
+    users.createUser('learner@editorAndPlayer.com', 'learner');
```

Comment:
Add space above. We want to separate each user's actions so it's easier to read.